HadesArchitect/grafana-cassandra-source
# Typescript Support

Added typescript, eslint, etc.

To test:
- review the changeset
- checkout branch `eslint`
- `docker run -v ${PWD}:/opt/gcds -w /opt/gcds node npm install`
- `docker run -v ${PWD}:/opt/gcds -w /opt/gcds node node_modules/eslint/bin/eslint.js src/*.js`

Expected output:
![image](https://user-images.githubusercontent.com/1742301/72244384-746cd700-35ee-11ea-8caa-d1b9812343ca.png)
